services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/core/errors.py

```diff
@@ -9,13 +11,11 @@ class ConfigurationError(ResourceUsageTrackerRuntimeError):
     msg_template: str = "Application misconfiguration: {msg}"
 
 
-class CreateServiceRunError(ResourceUsageTrackerRuntimeError):
-    msg_template: str = "Error during creation of new service run record in DB: {msg}"
+class MyHTTPException(HTTPException):
```

Comment:
Can we have a better name here? I know you want to keep it yours, but it's not helpful for who reeds it :)

Reply:
Tip2: if you are always going to use it with 404, just make it so that in he constructor the default is status is 404 so you don't have to specify it.
I see that in the tests you test for 404.